src/argilla/server/models/database.py

```diff
@@ -196,6 +195,10 @@ class MetadataProperty(DatabaseModel):
 
     __table_args__ = (UniqueConstraint("name", "dataset_id", name="metadata_property_name_dataset_id_uq"),)
 
+    @property
+    def type(self) -> MetadataPropertyType:
+        return self.settings["type"]
```

Comment:
The raw value in settings should be parsed, right?
```suggestion
        return MetadataPropertyType(self.settings["type"])
```

Reply:
The type conversion is not needed. But in that case, I would say to return an `str` instead of a `MetadataPropertyType`